iogt_users/tests.py

```diff
@@ -3,15 +3,19 @@
 from wagtail.core.models import Site
 
 from home.factories import SiteSettingsFactory, HomePageFactory
-from iogt_users.factories import UserFactory
+from iogt_users.factories import UserFactory, AdminUserFactory, GroupFactory
 from wagtail_factories import SiteFactory
 from questionnaires.factories import SurveyFactory
+from django.contrib.auth.models import Permission
 
 
 class PostRegistrationRedirectTests(TestCase):
     def setUp(self):
         self.user = UserFactory(has_filled_registration_survey=False)
-        self.admin_user = UserFactory()
+        self.admin_user = AdminUserFactory(has_filled_registration_survey=False)
+        self.admin_access_permission = Permission.objects.get(codename='access_admin')
+        self.group = GroupFactory()
+        self.group.permissions.add(self.admin_access_permission)
 
         Site.objects.all().delete()
```

Comment:
I think we've already established that this should not be necessary in any test.